README.md

```diff
@@ -6,6 +6,11 @@ OpenTelemetry community content.
 
 We are on [gitter](https://gitter.im/open-telemetry/community).
 
+## Community Meetings
+We host monthly community video conferences, alternating between times that work for Europe and Asia. These meetings are open to all contributors!
+
+Meeting invites can be accessed from our public Calendar ([web](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&ctz=America%2FLos_Angeles), [gCal](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9iNzllM2U5MGo3YmJzYTJuMnA1YW41bGY2MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t), [iCal](https://calendar.google.com/calendar/ical/google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com/public/basic.ics)), and we keep public notes in the agenda document available [here](https://docs.google.com/document/d/1sG3MAwHzYTlv5PrIBBpjZBTiShOpmzmPlLRe8NXZKnM/edit).
```

Comment:
re the agenda doc: that's the OC agenda doc, yes? (there is also an OT one) would it make sense to start fresh?

Reply:
Agreed, updating